### PR TITLE
my0u contest update

### PR DESCRIPTION
**Pull request by BenGardiner**

### CODE REVIEW

**Comment by kbembedded** [comment]
I don't have any issues with the change per se. There are some formatting and grammatical changes I'd request.

But I am curious the timeline to open up the contest in regards to the submission details. Do we instead want to make an online/printable form with those details to have people bring?

**Comment by BenGardiner** [comment]
> I don't have any issues with the change per se. There are some formatting and grammatical changes I'd request.

sure / shoot

> But I am curious the timeline to open up the contest in regards to the submission details. Do we instead want to make an online/printable form with those details to have people bring?

that's a great idea

**Comment by ShortTie** [comment]
"These all great!"? Meme speak I am not up on or typo?

"bypassing DRM" - Kind of self incriminating to state it that way. I mean we don't want to make it too easy for those looking to prosecute.

I am cool with making a printable form, as well as posting the requirements on the site. Do we want to allow a online "unofficial" submission, via social media with the same details? Not everyone can come, and I would love to encourage the sharing even if they don't travel to Vegas. As far as timing, I don't think this has to go out right away, the required info could be written in a drunken stupor as they dropped it off.

**Comment by kbembedded** [comment]
> I am cool with making a printable form, as well as posting the requirements on the site. Do we want to allow a online "unofficial" submission, via social media with the same details? Not everyone can come, and I would love to encourage the sharing even if they don't travel to Vegas. As far as timing, I don't think this has to go out right away, the required info could be written in a drunken stupor as they dropped it off.

That's a good point to have online and in-person submissions. In that case, I think having both something printable and submittable online would be great. If we're still cool with using google forms, it would probably be easy enough to slap something together that someone could fill out and submit in one go, or, fill it out, print it (no need to submit) and take it to con. 



> > I don't have any issues with the change per se. There are some formatting and grammatical changes I'd request.
> 
> sure / shoot

I'll pull up the PR tonight to verify they render and make some changes for you to consider.

**Comment by kbembedded** [comment]
Take a look at the changes I've and let me know if those work.

Before we go live with this, we probably want to figure out details on submissions (pre-made form, goog form, online submissions, etc.) though.

File: events/makeyourownuse.md [hunk 1]
```diff
@@ -3,17 +3,21 @@ title: Make Your 0wn Use Contest
 layout: page
 ---
 
-Have you taken your IoT toaster and created a remote activated fire-alarm tester? How about that old toy your kids (right, it was for the kids?) don't play with that now fuzzes your neighbor's drone? Or what about putting that con badge to good use? These all great! The Hardware Hacking Village is hosting a "Make Your 0wn Use" contest. This is the place to show off your skills at repurposing the electronics that surround and engulf us daily.
+Have you taken your IoT toaster and created a remote activated fire-alarm tester? How about that old toy your kids (right, it was for the kids?) don't play with that now fuzzes your neighbor's drone? Or what about putting that con badge to good use? The Hardware Hacking Village is hosting a "Make Your 0wn Use" contest. Submissions can be a solo or team based project that bend, mend, or repurpose any device and show others how it can be done.
```
Comment (BenGardiner) [inline events/makeyourownuse.md]: > bend, mend, or repurpose

👩‍🍳 💋

**Comment by BenGardiner** [comment]
I love the edits. Thank you, @kbembedded .

> Before we go live with this, we probably want to figure out details on submissions (pre-made form, goog form, online submissions, etc.) though.

I liked your idea of a google form and a printable option for bringing-in at DEF CON. I've executed a draft of that in the most recent changes.

**Comment by BenGardiner** [comment]
(rebased to resolve merge conflict)

**Comment by kbembedded** [comment]
Okay! All of that looks good, thanks!

Last question on this. Is this an in-person only contest or are online-only submissions accepted? I thought I asked earlier but it does not look like I did. If both, then we might want to clarify that in the web page (the PDF and form are fine).

File: events/makeyourownuse.md (hunk 1 above)
Comment (kbembedded) [reply events/makeyourownuse.md]: :D

**Comment by BenGardiner** [comment]
Good question (also asked by Tie). And yep we definitely want to enable as much sharing of awesome my0u as possible. But we should probably limit the _competition to in-person.

I updated the event text, form and printable.

**Pull request merged**